### ISSUE TITLE
TRACK: Sprint review 9.1.24

### ISSUE THREAD

**Issue by cmarooney-stfc** [pace-neutrons/Horace#1470]
(a) Major items in progress or planned

ALEX:

 - [ ] Filebacked:                #1302 by 18.12.23 possibly earlier - EPIC modified, most things finished/under review or minor side issues
 - [ ] Tutorial validation:    #1314 start 2.1.24 latest, end 18.1.24. Plus any tickets arising. At 8.1.24 20% complete.
  (based on Alex local copy, check if current given multifit comments, check Sharepoint, circulate, cm to create sharepoint folder, by 10.1.24)
 - [ ] Spherical projection with Tobyfit #1040 23.1.24 to 23.2.24 (latest, could be 15.1.24) depending on scoping exercise TOBY ALEX (DUC?) TO DISCUSS 8.1.24 do first to inform scope of next 2:
ALEX to discuss with Duc on whether this meets user needs, soon (based on Christian's uncertainties, get representative dataset) - image of what Christian wants and what Christian got (+rb-no. if available)
 - [ ] Cylindrical projections (5d) insert TICKETS 8.1.24 - 22.1.24
 - [ ] focussed projections (5d) insert TICKETS 8.1.24 - 22.1.24
 - [ ] Horace help issue needs ticketing to continue having Fan work on this

JACOB

 - [ ] Tobyfit memory: #760 in progress, by 8.1.24 PR await submission, nb still not filebacked, may need pageop implementation for 4-way zip plus extra optimisations. Ensure this solves Duc's issue.
- [ ] Profiling:  #676 BACKGROUND ONGOING 2.1.24-2.2.24 have fake data, need real data, want known time-hoggers JACOB TOBY TO DISCUSS w/c 15.1.24 (profilng, parallelization)
- [ ] Fix CI, need ticket, SL7 nodes retiring. (2d) Add additional Matlab versions to new R8 nodes.
- [x] Jacob #1172 JENKINS ONLY -
- [ ] Review documentation changes (how do we do this?) still need to ask Excitations/David eg at Excitations awayday
- [ ] email from Boothroyd post-doc -  need to investigate their problems
- [x] pace-builder a/c needs 2fa - sort within week

JW notes: 17/1/24 
- David harangued will schedule and arrange documentation review within Excitations. Plan akin to example script development assigning sections to individuals.
- Viviane's (Boothroyd's post-doc) IDAaaS workspace expired over New Year's break email sent requesting new one
- CI pipelines set up and "working" but lacking `cppcheck`, email sent to Alan Kyffin. Followup when time has passed to chase. 

Update 18/1/24:
- Alan has installed more MATLAB versions and newest cppcheck

CHRIS

Detector tickets 
 - [ ] #1432 and earlier (#1350, #1298, #1338, #1398)

Container tickets
 - [ ] #1157
 - [ ] #1078 
 - [ ] Note on hashing issues, Toby to ticket

TOBY

 - [ ] Talk to Duc about info in nxspe; DUC/TOBY Chase Mantid team detector info (script failing):  Duc says Merlin and Maps raw files have info -  Toby to browse hdf5 output - need to get this into nxspe, raise Mantid epic - Toby and Jacob talk to Duc
 - [ ] Toby ticket hashing issues, yes, Chris to add.
 - [ ] troubleshoot, ongoing, background
 - [ ] run challenging cases (generate them) with H4 e.g. cut-sym
 - [ ] Radu Epic - TGP review and comment on JW tickets - this sprint

____

(b) Suggestions for next areas to do

 - [ ]  Parallelization #1253, includes:-
 - [ ] #1203 struggled to reproduce but needs to be confirmed & fixed, JW HIGH PRIORITY early Jan
 - [ ] Performance - #1419 Tentative for JW 9.1.24
 - [ ] Resolution function plot issues - #1408  Toby to deal with BACKGROUND, Jan 24
- [ ] ??? #922 Toby progressing; required for mgenie detector tools (???)#921

---

(c) New issues prioritised up to 28.11.23

Filebacked

 - [ ] #1404

Performance

- [ ] #1372, #1364 WAIT ON BENCHMARKING

Tobyfit/Detectors

 - [ ] #1430
 - [ ] #1419 
 - [ ] #1398 

Parallel

 - [ ] #1393 [POST-CONSTRUCTION, SCARF check of fixes to IDAaaS #12No tickets92, #1391]

Tests

No open tickets

Filebacked epic #1302 
 - remaining time estimated as 3 weeks = 19.12.23

----------------------

(e) Previous issues still outstanding

*Think these are largely dealt with above*
See previous sprint review 12.12.23 for details when the main issues are cleared

**Comment by cmarooney-stfc** [comment]
closed as superseded by #1502
